ote/src/cljs/ote/views/route/stop_sequence.cljs

```diff
@@ -61,82 +61,106 @@
         [common/help (tr [:route-wizard-page :stop-sequence-custom-dialog-help])]]])))
 
 (defn- route-map [e! route]
-  (r/create-class
-   {:component-did-mount
-    (fn [this]
-      (let [deleting? (atom false)
-            ^js/L.Map
-            m (aget this "refs" "stops-map" "leafletElement")]
+  (let [dc (atom nil)]
+    (r/create-class
+      {:component-did-mount
+        (fn [this]
+          (let [deleting? (atom false)
+               ^js/L.Maps
```

Comment:
extra `s` in type, `L.Map` -> `L.Maps` mistake?

Reply:
yep. thanx